content/courses/advanced-ada/parts/control_flow/exceptions.rst

```diff
@@ -865,20 +867,64 @@ checks fails and raises a :ada:`Storage_Error` exception.
         - Complete section!
 
 
-..
-    TO BE DONE:
 
-    :ada:`Tasking_Check`
-    ~~~~~~~~~~~~~~~~~~~~
+.. _Adv_Ada_Tasking_Check:
 
-    .. admonition:: In the Ada Reference Manual
+:ada:`Tasking_Check`
+~~~~~~~~~~~~~~~~~~~~
 
-        - :arm22:`11.5 Suppressing Checks <11-5>`
+The :ada:`Tasking_Check` ensures that all tasks have been activated
+successfully and that no terminated task is called. If the check fails, a
+:ada:`Tasking_Error` exception is raised.
 
-    .. todo::
+.. note::
 
-        - Complete section!
+   This concept was introduced in Ada 2022. It was created to group all checks
+   that might raise the :ada:`Tasking_Error` exception.
```

Comment:
I don't understand -- this check has existed since Ada 83. What am I missing?

Reply:
Yes, the check itself already existed, but it didn't have a specific name. That was corrected in Ada 2022, see this AI: http://www.ada-auth.org/cgi-bin/cvsweb.cgi/ai12s/ai12-0309-1.txt?rev=1.5&raw=N

Reply:
OK, thanks.